client/my-sites/plans-features-main/test/index.jsx

```diff
@@ -309,6 +311,70 @@ describe( 'PlansFeaturesMain.getPlansForPlanFeatures() with tabs', () => {
 			comp.find( 'SegmentedControlItem[path="?customerType=personal"]' ).props().selected
 		).toBe( false );
 	} );
+
+	test( 'Should highlight proper popular plan for blog site type', () => {
```

Comment:
What do you think about explicitly stating the `TYPE_PERSONAL` plan in this test description (and so on, for the others)?
I don't feel too strongly, so I'm happy for it to be left as is, just a thought :)

Reply:
Yeah I think this is a good idea, better when you're scanning through test titles.